lib/quicly.c

```diff
@@ -3923,12 +3880,10 @@ int quicly_accept(quicly_conn_t **conn, quicly_context_t *ctx, struct sockaddr *
     egress_cipher = (struct st_quicly_cipher_context_t){NULL};
     (*conn)->crypto.handshake_properties.collected_extensions = server_collected_extensions;
 
-    LOG_CONNECTION_EVENT(*conn, QUICLY_EVENT_TYPE_ACCEPT, VEC_EVENT_ATTR(DCID, packet->cid.dest.encrypted),
-                         VEC_EVENT_ATTR(SCID, packet->cid.src));
-    LOG_CONNECTION_EVENT(*conn, QUICLY_EVENT_TYPE_CRYPTO_DECRYPT, INT_EVENT_ATTR(PACKET_NUMBER, pn),
-                         INT_EVENT_ATTR(LENGTH, payload.len));
-    LOG_CONNECTION_EVENT(*conn, QUICLY_EVENT_TYPE_QUICTRACE_RECV, INT_EVENT_ATTR(PACKET_NUMBER, pn),
-                         INT_EVENT_ATTR(LENGTH, payload.len), INT_EVENT_ATTR(ENC_LEVEL, QUICLY_EPOCH_INITIAL));
+    QUICLY_PROBE(ACCEPT, *conn, now, QUICLY_PROBE_HEXDUMP(packet->cid.dest.encrypted.base, packet->cid.dest.encrypted.len),
+                 QUICLY_PROBE_HEXDUMP(packet->cid.src.base, packet->cid.src.len));
+    QUICLY_PROBE(CRYPTO_DECRYPT, *conn, now, pn, payload.base, payload.len);
+    QUICLY_PROBE(QUICTRACE_RECV, *conn, now, pn, payload.len, QUICLY_EPOCH_INITIAL);
```

Comment:
QUICTRACE_RECV records the length of the plaintext payload, whereas QUICTRACE_SEND logs the length of the QUIC packet. Is this the intended behavior?

Reply:
The length of the received packet is not used by quictrace. I wonder if this is just leftover... feel free to remove the payload length from the receive event entirely.

Reply:
Removed in b9c9c1f.